### ISSUE TITLE
[QUESTION] MK3.5 with the 0.9 steppers

### ISSUE THREAD

**Issue by Lino77** [prusa3d/Prusa-Firmware-Buddy#3729]
Please, before you create a new question, please make sure you searched in open and closed issues and couldn't find anything that matches.

**Leave your comments or questions here.**
Is it possible to use the MK3.5 with the 0.9 steppers

**Comment by danopernis** [comment]
No, at the moment it is not possible to use MK3.5 with 0.9° steppers.
